lib/kitchen/document.rb

```diff
@@ -177,6 +177,18 @@ def modified_id_to_paste(original_id)
       end
     end
 
+    # Copies an ID, returning a unique ID
+    #
+    # @param id [String]
+    #
+    def copy_id(id)
+      return nil if id.nil?
+      return '' if id.blank?
+
+      @next_paste_count_for_id[id] ? @next_paste_count_for_id[id] += 1 : @next_paste_count_for_id[id] = 1
```

Comment:
is there a more syntactically nice way of doing this?

Reply:
May not be necessary per above, but there might be a way to make this better using hash default values